### DISCUSSION TITLE
How to select multiple images on IOS 13 or below?

### DISCUSSION THREAD

**Discussion by lamlengend98** [react-native-image-picker/react-native-image-picker#1869] [Q&A]

**Comment by lukebrandonfarrell** [comment]
It's not possible with PHPhotoPicker, you may want to build an implementation where you keep opening the picker in a loop until the requested amount of images is picked, OR pull in another library specifically for iOS 13. That is the best solution I can think of, although I'd be interested in starting a conversation around this, as it's also a problem for some users in our product.
